sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -320,12 +321,18 @@ class SQLAppStatusListener(
   private def onExecutionEnd(event: SparkListenerSQLExecutionEnd): Unit = {
     val SparkListenerSQLExecutionEnd(executionId, time) = event
     Option(liveExecutions.get(executionId)).foreach { exec =>
-      exec.metricsValues = aggregateMetrics(exec)
       exec.completionTime = Some(new Date(time))
-      exec.endEvents += 1
       update(exec)
```

Comment:
As a side-effect, this PR seems to increase the number of invocation `update(exec)` exactly twice. Is this okay?

Reply:
Yes.

Reply:
Got it. Thank you for confirming.